DEPS

```diff
@@ -441,6 +443,31 @@ deps = {
      'condition': 'download_android_deps',
      'dep_type': 'cipd',
    },
+
+   'src/fuchsia/sdk': {
+     'packages': [
+       {
+        'package': 'fuchsia/sdk/core/${{platform}}',
+        # Get the SDK from https://chrome-infra-packages.appspot.com/p/fuchsia/sdk/core at the 'latest' tag
+        'version': 'jlQvNeRMq6X81_VYiFI_Ol311YCXak0xACebeb8f6TcC'
```

Comment:
Do mac and linux have the same version string?

Reply:
Oops. No.